Makefile.toml

```diff
@@ -354,9 +366,6 @@ cargo build \
 '''
 ]
 
-[tasks.world-packages]
-alias = "build-packages"
-
 [tasks.world-variant]
 alias = "build-variant"
```

Comment:
We should clean up the other "world" aliases; they were for muscle-memory compatibility for the Makefile from aeons past.